### PR TITLE
new site

### PR DESCRIPTION
**Pull request by martinraison**
@stkem @2is10  Checking that this works. I'll probably have to cleanup a few more things when I move the blog and so on.

### CODE REVIEW

**Comment by stkem** [comment]
lgtm, I guess :-)

File: kifi-backend/modules/shoebox/angular-black/src/layout/rightCol/rightCol.tpl.html [hunk 1]
```diff
@@ -73,7 +73,7 @@ <h2 class="kf-footer-header">Info and support</h2>
         <a class="kf-footer-item-link updates-features" href="https://www.kifi.com/terms">Terms</a>
       </li>
       <li>
-        <a class="kf-footer-item-link updates-features" href="http://www.42go.com">About us</a>
+        <a class="kf-footer-item-link updates-features" href="http://www.kifi.com/about_us/mission.html">About us</a>
```
Comment (2is10) [inline kifi-backend/modules/shoebox/angular-black/src/layout/rightCol/rightCol.tpl.html:76]: Are we serving some pages over http:// (non-SSL) now?

File: kifi-backend/modules/shoebox/conf/shoebox.routes [hunk 2]
```diff
@@ -196,6 +196,8 @@ GET     /profile                    @com.keepit.controllers.website.HomeControll
 GET     /find                       @com.keepit.controllers.website.HomeController.home
 GET     /blog                       @com.keepit.controllers.website.HomeController.blog
 
+GET     /about_us/*file             controllers.Assets.at(path="/public/about_us", file)
```
Comment (2is10) [inline kifi-backend/modules/shoebox/conf/shoebox.routes:199]: Would Play barf if we made this `/about/*file` instead? Much cleaner than `/about_us/`, in my opinion.

File: kifi-backend/modules/shoebox/public/about_us/join_us.html [hunk 3]
```diff
@@ -0,0 +1,286 @@
...
+
+<div class="buttons_row">
+    <a href="mailto:careers@kifi.com?subject=I'm your mobile engineer!" target="_blank" class="button blue filter">That's Me</a>
+</div>
+
+<hr>
+
+<a id="backend"><h3 class="blue">Backend Engineer</h3></a>
+
+<p>Role: Growing and scaling up FortyTwo’s backend. At this stage of the company it means going all the way from building clusters of services, optimizing concurrent processing, building data pipelines, working with various types of databases and implementing devops automations. We are building our systems on using technologies like <a href="http://www.42go.com/culture.html#technologies">Scala, Play, Akka, ZooKeeper and Guice</a>.</p>
```
Comment (2is10) [inline kifi-backend/modules/shoebox/public/about_us/join_us.html:159]: bad link to 42go.com

**Comment by 2is10** [comment]
cool, thanks for doing

File: kifi-backend/modules/shoebox/angular-black/src/layout/rightCol/rightCol.tpl.html (hunk 1 above)
Comment (martinraison) [inline kifi-backend/modules/shoebox/angular-black/src/layout/rightCol/rightCol.tpl.html:76]: non intentional - thanks for the catch!

**Pull request merged**